Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM rocker/r-ver:4.0.4
+FROM rocker/r-ver:4.4.1
 
 # DeGAUSS container metadata
 ENV degauss_name="census_block_group"
-ENV degauss_version="0.6.0"
+ENV degauss_version="1.0.0"
```

Comment:
As far as I'm aware, this is where the version metadata is set. Am I missing anything else if I want to update the version number?

Reply:
this is it!